manifests/init.pp

```diff
@@ -4,74 +4,78 @@
 #
-# $pulpcore_postgresql_password::       Password of the Pulpcore PostgreSQL database.
+# $pulpcore_postgresql_password::              Password of the Pulpcore PostgreSQL database.
 #
-# $pulpcore_postgresql_db_name::        Name of the Pulpcore database in PostgreSQL.
+# $pulpcore_postgresql_db_name::               Name of the Pulpcore database in PostgreSQL.
 #
-# $pulpcore_postgresql_ssl::            Enable SSL connection to the Pulpcore PostgreSQL database. Only meaningful for external/unmanaged DB.
+# $pulpcore_postgresql_ssl::                   Enable SSL connection to the Pulpcore PostgreSQL database. Only meaningful for external/unmanaged DB.
 #
-# $pulpcore_postgresql_ssl_require::    Configure Pulpcore to require an encrypted connection to the PostgreSQL database.
+# $pulpcore_postgresql_ssl_require::           Configure Pulpcore to require an encrypted connection to the PostgreSQL database.
 #
-# $pulpcore_postgresql_ssl_cert::       Path to SSL certificate to use for Pulpcore connection to PostgreSQL database.
+# $pulpcore_postgresql_ssl_cert::              Path to SSL certificate to use for Pulpcore connection to PostgreSQL database.
 #
-# $pulpcore_postgresql_ssl_key::        Path to key file to use for Pulpcore connection to PostgreSQL database.
+# $pulpcore_postgresql_ssl_key::               Path to key file to use for Pulpcore connection to PostgreSQL database.
 #
-# $pulpcore_postgresql_ssl_root_ca::    Path to the root certificate authority to validate the certificate supplied by the PostgreSQL database server.
+# $pulpcore_postgresql_ssl_root_ca::           Path to the root certificate authority to validate the certificate supplied by the PostgreSQL database server.
 #
-# $pulpcore_worker_count::              Number of pulpcore workers. Defaults to 8 or the number of CPU cores, whichever is smaller.
-#                                       Enabling more than 8 workers, even with additional CPU cores available, likely results in performance
-#                                       degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should be done
-#                                       incrementally with benchmarking at each step to determine an optimal value for your deployment.
+# $pulpcore_worker_count::                     Number of pulpcore workers. Defaults to 8 or the number of CPU cores, whichever is smaller.
+#                                              Enabling more than 8 workers, even with additional CPU cores available, likely results in performance
+#                                              degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should be done
+#                                              incrementally with benchmarking at each step to determine an optimal value for your deployment.
+#
+# $pulpcore_content_gunicorn_worker_timeout::  Gunicorn worker timeout in seconds for the pulpcore-content.service
```

Comment:
In the pulpcore module we use `service` instead of `gunicorn`, I think I like the use of `service` better and its provides consistency with the base module.